kie-archetypes/kie-kjar-archetype/src/main/resources/archetype-resources/global/customeditors.json

```diff
@@ -0,0 +1,3 @@
+{ "editors":{
```

Comment:
what is this for?

Reply:
it's for legacy designer. this so people can see an example how to create custom editors for parameters (every parameter in old designer can have a custom editor)